### ISSUE TITLE
how do I generate 16 character length secret key consistently?

### ISSUE THREAD

**Issue by vafada** [yeojz/otplib#201]
**Describe the bug**

When generating a `secret` via `const secret = authenticator.generateSecret()`

the length of `secret` varies on every run.

**To Reproduce**
1.

```
const secret = authenticator.generateSecret()
console.log(secret);
console.log(secret.length);
```

2. Run step 1 multiple times, `secret.length` will be different for every run

**Expected behavior**

I expect the secret generated of have a consistent length of 16

**Details (please provide any relevant information):**

- otplib version: v12
- os: linux
- browser:
- node / yarn: node v.12.13.0
- frameworks (if any):

**Additional context**

the website ALWAYS generates a 16 character length string:

https://github.com/yeojz/otplib/blob/master/website/src/components/SecretStore.jsx#L19-L21

**Comment by yeojz** [comment]
Thanks for the report. Will look into it.

At a glance, it looks like it is highly likely due to the thirty-two implementation.

**Comment by yeojz** [comment]
Thanks. Found the area of concern and will be addressed in #202
